### PR TITLE
chores: Added README.d and basic CI.

### PR DESCRIPTION
**Pull request by k-avy**
Fixes issue: #3

### CODE REVIEW

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# SPDX-FileCopyrightText: 2023 Kavya Shukla <kavyuushukla@gmail.com>
+# X-License-Identifier: GPL-2.0
```
Comment (GMishx) [inline .github/workflows/go.yml:2]:
```suggestion
# SPDX-License-Identifier: GPL-2.0-only
```
Please use the new license identifier. And update other files as well in a separate PR.

File: README.md [hunk 2]
```diff
@@ -3,3 +3,35 @@
...
+cd LicenseDb
+```
+
+- Build the project using following command.
+
+```bash
+go build ./cmd/laas
+```
+
+- Run the executable in your vscode terminal.
```
Comment (GMishx) [inline README.md:27]:
```suggestion
- Run the executable.
```

**Review by GMishx — CHANGES_REQUESTED** [review]

**Comment by avinal** [comment]
Please squash the commits into one.

**Review by avinal — CHANGES_REQUESTED** [review]
Please squash the commits into one.

**Comment by avinal** [comment]
@GMishx we need to enable GitHub Actions

**Review by GMishx — APPROVED** [review]

**Pull request merged**